docs/docsite/rst/community/development_process.rst

```diff
@@ -168,7 +168,7 @@ The toplevel keys (bugfixes, major_changes, and so on) are defined in the
 `config file <https://github.com/ansible/ansible/blob/devel/changelogs/config.yaml>`_ for our release note tool. Here are the valid sections and a description of each:
 
 **major_changes**
-    Major changes to Ansible itself. Generally does not include module or plugin changes.
+  Major changes to Ansible itself. Generally does not include module or plugin changes.
 
 **minor_changes**
   Minor changes to Ansible, modules, or plugins. This includes new features, new parameters added to modules, or behavior changes to existing parameters.
```

Comment:
Should we update this to reflect the fact that most modules and plugins now reside in collections?

Reply:
please take a look at the last commit, thanks!